app/graphql/types/query_type.rb

```diff
@@ -29,6 +29,7 @@ class QueryType < Types::BaseObject
 
     field :digital_objects, DigitalObjectInterface.results_type, null: true, extensions: [Types::Extensions::SolrSearch, Types::Extensions::MapToDigitalObjects] do
       description "List and searches all digital objects"
+      argument :order_by, Inputs::DigitalObject::OrderByInput, required: false, default_value: { field: 'score', direction: 'desc' }
```

Comment:
We initially had score-based default sorting in Hyacinth a long time ago, but users ended requesting and preferring title-based default sorting instead so we switched to that.

Reply:
Do you think that should be the default for the API and UI?

Reply:
If we can separate the two, I would use the Hyc2 default in the UI.

Reply:
I expect that from the UI we would always set the sort parameters, so we can definitely specify a different sort value for the UI and API.

Reply:
@elohanlon: I am going to merge this request, but if you think the API default should be different let me know and I can change it in a subsequent request. I noted that the default for the UI should be title sort here: https://issues.cul.columbia.edu/browse/HYACINTH-571

Reply:
@cgalarza Default in UI-only sounds good to me!  Thank you for the ticket.